webapp/channels/src/components/advanced_text_editor/scheduled_post_indicator/scheduled_post_indicator.tsx

```diff
@@ -36,18 +38,10 @@ export default function ScheduledPostIndicator({location, channelId, postId, rem
     // when opening from center channel.
     const id = location === Locations.RHS_COMMENT ? postId : channelId;
     const scheduledPostData = useSelector((state: GlobalState) => showChannelOrThreadScheduledPostIndicator(state, id));
-    const match: match<{team: string}> = useRouteMatch();
-
-    const scheduledPostLinkURL = `/${match.params.team}/scheduled_posts`;
 
-    const link = useMemo(() => (
-        <NavLink to={scheduledPostLinkURL}>
-            <FormattedMessage
-                id='scheduled_post.channel_indicator.link_to_scheduled_posts.text'
-                defaultMessage='See all scheduled messages'
-            />
-        </NavLink>
-    ), [scheduledPostLinkURL]);
+    const basePath = useSelector(getBasePath);
+    const currentTeam = useSelector(getCurrentTeam);
```

Comment:
```suggestion
    const currentTeamName = useSelector((state) => getCurrentTeam(state)?.name);
```

This avoids the component to re-render over irrelevant changes in the team object.

Reply:
Done.,